### ISSUE TITLE
«po»-initial compounds

### ISSUE THREAD

**Issue by uakci** [toaq/dictionary#87]
Toaq has many compounds beginning in **po–**, especially the personal possessive pronouns — **pojī, pohō, pomȳı.**

My question is, why do we have to have them like this if they break the almost-universal rule of compounds which says that a compound form of a phrase is that phrase with its lexemes reversed? That is to say, why don't we use **jıpō, hopō, myıpō** in analogy with other compounds?

~

**Comment by Ilmen-vodhr** [comment]
**po** is a particle and doesn't have the regular contentive syntax, so it's not so weird that it also behaves differently in root compounds. But in the end the choice is probably arbitrary there.

**Comment by uakci** [comment]
That is right, and since it's room for an arbitrary choice, let's make that choice good :)

(Though I'm not particularly bothered by the current state of affairs, so let me add a ‘petty’ tag to reflect this.)
